src/routes/concepts/context.mdx

```diff
@@ -10,12 +10,11 @@ Context provides a way to pass data through the component tree without having to
 When you have a large [component tree](/concepts/components/basics#component-trees) that requires state to be shared, context can be used.
 Context can be employed to avoid [prop drilling](/concepts/components/props#prop-drilling), which is the practice of passing props through intermediate elements without using them directly.
 
-If you only want to avoid passing some props through a few levels, component composition is often a simpler solution than context.
+If you only want to avoid passing "some" props through a "few" layers, adjusting your component hierarchy may be an easier solution, when applicable.
+[Signals](/concepts/signals), when injected directly into a component that needs them, will always be the simplest and most explicit solution.
```

Comment:
```suggestion
[Signals](/concepts/signals) are often the simplest solution since they can be imported directly into the components that need them.
```